### PR TITLE
Add open-dir-in-finder extension

### PR DESCRIPTION
**Pull request by joshrad-dev**
## Description

This extension allows a user to quickly open a directory in finder by typing in the path.

## Type of change

- New extension

## Screencast

https://user-images.githubusercontent.com/62785552/137667150-478665c6-8e3f-4fe5-92fb-be65ff96294d.mov

## Checklist

- [✅] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)

### CODE REVIEW

File: extensions/open-dir-in-finder/src/index.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+const markdown = `
+# Markdown
+
+Simply render markdown, including:
+- Headings and blockquotes
+- **Bold** and _italics_ emphasis
+- Code blocks and \`inline code\`
+
+For more, check out the [CommonMark](https://commonmark.org) spec.
+`;
```
Comment (thomaspaulmann) [inline extensions/open-dir-in-finder/src/index.tsx:13]: Looks like this isn't used anymore. Could you remove? And maybe also clean up the imports.

File: extensions/open-dir-in-finder/src/index.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+export default function main() {
+  const [text, setText] = useState("");
+  return <List onSearchTextChange={(t) => { setText(t) }} searchBarPlaceholder="Directory...">
+    <List.Item title={`Open in Finder:`} subtitle={text} icon={Icon.Finder} actions={
+    <ActionPanel>
+      <ActionPanel.Item title="Open in Finder" onAction={() => {exec(`open ${text}`, (err, stdout, stderr) => {
```
Comment (thomaspaulmann) [inline extensions/open-dir-in-finder/src/index.tsx:20]: Did you try using the `ShowInFinder` action?

File: extensions/open-dir-in-finder/package.json [hunk 3]
```diff
@@ -0,0 +1,34 @@
+{
+  "name": "open-dir-in-finder",
+  "title": "Open in Finder",
```
Comment (thomaspaulmann) [inline extensions/open-dir-in-finder/package.json:3]: Can we change this title to something more descriptive? It's what shows up in the store, so people should understand what they install...

```suggestion
  "title": "Open in Finder",
```

File: extensions/open-dir-in-finder/package.json [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  "description": "Opens directory in finder",
+  "icon": "Finder-Icon.png",
+  "author": "jawshoeadan",
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Open in Finder",
+      "subtitle": "",
+      "description": "Template to run a script without any view",
```
Comment (thomaspaulmann) [inline extensions/open-dir-in-finder/package.json:13]: Update? ;-)

**Comment by jawshoeadan** [comment]
I was wondering, do you think I should rewrite this as a script command instead of an extension? I think it may look cleaner if I do.

**Comment by thomaspaulmann** [comment]
> I was wondering, do you think I should rewrite this as a script command instead of an extension? I think it may look cleaner if I do.

Well, no need to rewrite it. It's already a [script command](https://github.com/raycast/script-commands/blob/805ce902804376bb92371ae35d4a974eba4a6aeb/commands/navigation/open-finder.sh) ;-) Should we then close this PR?

**Pull request closed without merging**